app/src/components/items/item.tsx

```diff
@@ -40,13 +41,17 @@ const Item = ({ manifest, item, canvasIndex }: ItemProps) => {
             <Heading level="h2">{item.title}</Heading>
           </>
         )}
-        <Text marginTop="m">
-          <Icon name="errorOutline" size="medium" /> Our collections include
-          some content that may be harmful or dificult to view.{" "}
-          <Link href="https://digitalcollections.nypl.org/about#nypl_harmful_content_statement">
-            Learn more.
-          </Link>{" "}
-        </Text>
+        {/* TODO: horizontally align Icon with Text with breakpoints */}
```

Comment:
idk if this is a nitpick or if this will be a VQA update but maybe I can remove this TODO